doc/classes/MultiplayerPeerCustom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="MultiplayerPeerCustom" inherits="NetworkedMultiplayerPeer" version="3.5" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A [NetworkedMultiplayerPeer] implementation that can be controlled from a script.
+	</brief_description>
+	<description>
+		A [NetworkedMultiplayerPeer] implementation that can be used as a [member MultiplayerAPI.network_peer] and controlled from a script.
+		Its purpose is to allow adding a new backend for the high-Level multiplayer API without needing to use GDNative.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="deliver_rpc">
+			<return type="void" />
+			<argument index="0" name="buffer" type="PoolByteArray" />
+			<argument index="1" name="from_peer_id" type="int" />
+			<description>
+				Deliver an RPC to the local [MultiplayerAPI].
+				When your script receives RPC data from other peers over the network, passing it to this method will execute the RPC locally.
+			</description>
+		</method>
+		<method name="initialize">
+			<return type="void" />
+			<argument index="0" name="self_peer_id" type="int" />
+			<description>
+				Initialize the peer with the given [code]peer_id[/code] (must be between 1 and 2147483647).
+			</description>
+		</method>
+		<method name="set_connection_status">
+			<return type="void" />
+			<argument index="0" name="connection_status" type="int" enum="NetworkedMultiplayerPeer.ConnectionStatus" />
+			<description>
+				Set the state of the connection. See [enum NetworkedMultiplayerPeer.ConnectionStatus].
+			</description>
+		</method>
+		<method name="set_max_packet_size">
+			<return type="void" />
+			<argument index="0" name="max_packet_size" type="int" />
+			<description>
+				Set the max packet size that this peer can handle.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="refuse_new_connections" type="bool" setter="set_refuse_new_connections" getter="is_refusing_new_connections" overrides="NetworkedMultiplayerPeer" default="false" />
+		<member name="transfer_mode" type="int" setter="set_transfer_mode" getter="get_transfer_mode" overrides="NetworkedMultiplayerPeer" enum="NetworkedMultiplayerPeer.TransferMode" default="2" />
+	</members>
+	<signals>
+		<signal name="rpc_generated">
+			<argument index="0" name="peer_id" type="int" />
+			<argument index="1" name="buffer" type="PoolByteArray" />
+			<argument index="2" name="transfer_mode" type="int" />
+			<description>
+				Emitted when the local [MultiplayerAPI] generates an RPC.
```

Comment:
as above, should probably mention this is raw bytes, and might not be an RPC (e.g. multiplayer.`send_bytes`).